### ISSUE TITLE
Add support for multiple recipient with one push message

### ISSUE THREAD

**Issue by mbordia-eightfold** [expo-community/expo-server-sdk-python#52]
As per this example in [expo documentation](https://docs.expo.dev/push-notifications/sending-notifications/#:~:text=%7D%2C%0A%20%20%7B-,%22to%22,-%3A%20%5B%0A%20%20%20%20%20%20%22ExponentPushToken) `to` attribute in PushMessage should expect `string[]` as well allowing support for multiple recipient for a push message.

There are few places in the codebase which results in the error when we pass list of expo tokens in `to` attribute:
1. [Expects to attribute to be string only](https://github.dev/expo-community/expo-server-sdk-python/blob/b39689985eee7faa433ae0ac18b992dab912a0ec/exponent_server_sdk/__init__.py#L326)
2. [Throws error even if response is expected response](https://github.dev/expo-community/expo-server-sdk-python/blob/b39689985eee7faa433ae0ac18b992dab912a0ec/exponent_server_sdk/__init__.py#L371)

I think proper implementation for this would either require :
a. removal of the assumption that 1 push message corresponds to single recipient 
b. or converting single PushMessage into multiple duplicate PushMessages.

**Comment by julianpacker** [comment]
Good point. I never noticed that. I think doing a is the ideal solution, seems more efficient. Let me review that PR
